bindings/pyroot_experimental/pythonizations/test/rvec.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+import ROOT
+
+
+class RVec(unittest.TestCase):
+    fundamental_types = ['int', 'unsigned int', 'long', 'long long', 'Long64_t', 'unsigned long',
+                         'unsigned long long', 'ULong64_t', 'float', 'double']
+    extra_types = ['bool']
+
+
+    def test_iteriter(self):
+        '''
+        Test the iteration over the iterator of an iterator
+
+        This breaks if __iter__ or tp_iter is not defined for the iterator and causes
+        issues, e.g., in comparison to numpy arrays.
+        '''
+
+        for dtype in self.fundamental_types + self.extra_types:
+            rvec = ROOT.VecOps.RVec[dtype](3)
+            it = iter(rvec)
+            it = iter(it)
```

Comment:
I think you wanted to save it in another variable `it2 = iter(it)` and then compare both in the assert?

Reply:
Oh, thanks :D That's why we do reviews :see_no_evil: